app/src/main/java/com/xa/xpensauditor/MonthWiseVis.java

```diff
@@ -47,8 +50,9 @@ public class MonthWiseVis extends AppCompatActivity {
     private Button Vis;
     private String Tid;
     int tot=0;
-
+    int ii=0;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '=' is not preceded with whitespace.